### DISCUSSION TITLE
Viewing the Bestiary

### DISCUSSION THREAD

**Discussion by evensong-sys** [Obsidian-TTRPG-Community/fantasy-statblocks#98] [Q&A] [answered]
Would anyone be able to help me understand how to view the bestiary using Dataview? I've looked through documentation for this and for dataview, tried reading some stuff on javascript, but still don't understand how to make it work. I've tried using (with the ``` things before and after):
dataview
const creatures = bestiary.values();

and:
dataviewjs
const creatures = bestiary.values();

Obviously, neither worked, but I don't know what else I need to include with it to get a list of creatures in my bestiary.

**Comment by kylomax** [comment, marked as answer]
You should be using `window.bestiary`
For instance, here's a codeblock that gets a monster and displays it in the note:

```dataviewjs
const bestiary = window.bestiary;
dv.span(bestiary.get("Ancient Black Dragon"));
```

do note that it must be in a `dataviewjs` code block.
